editor/js/editor-libs/telemetry.js

```diff
@@ -0,0 +1,68 @@
+import { postParentMessage } from "./events.js";
+
+const ACTION_COUNTS_KEY = "action-counts";
+
+function readActionsCounts() {
+  try {
+    const value = JSON.parse(localStorage.getItem(ACTION_COUNTS_KEY));
+    if (value && value.href === window.location.href && value.counts) {
+      return value.counts;
+    }
+  } catch (e) {
+    window.console.warn("Unable to read action counts from localStorage", e);
+  }
+
+  return {};
+}
+
+function persistActionCounts(counts) {
+  try {
+    localStorage.setItem(
+      ACTION_COUNTS_KEY,
+      JSON.stringify({
+        href: window.location.href,
+        counts,
+      })
+    );
+  } catch (e) {
+    window.console.warn("Unable to write action counts to localStorage", e);
+  }
+}
+
+const actionCounts = readActionsCounts();
+let lastAction = null;
+
+/**
+ * @param {string} key
+ * @param {boolean} once
+ */
+export function recordAction(key, deduplicate = false) {
+  actionCounts[key] = actionCounts[key] ?? 0;
+
+  if (deduplicate && key === lastAction) {
+    return;
+  } else {
+    lastAction = key;
+  }
+
+  let source = `${key} -> ${actionCounts[key]}`;
```

Comment:
We can use `const` here.

Reply:
Resolved via eab1078fc2068c7ffc3e8967d4137499b6099a51.